package.json

```diff
@@ -60,7 +60,7 @@
     "@typescript-eslint/eslint-plugin": "^4.28.3",
     "@typescript-eslint/parser": "^4.28.3",
     "chai": "^4.3.4",
-    "eslint": "^7.30.0",
+    "eslint": "^7.32.0",
```

Comment:
Why is this necessary?

Reply:
Not yet sure as even if I had eslint 7 already installed I got an error about `es2021` not being valid in current config, upgrading did fix it but I am not sure which version I used before that.